lib/developer_portal/app/views/developer_portal/main_layout.html.liquid

```diff
@@ -22,6 +22,9 @@
     {{ '/javascripts/vendor/colorbox/jquery.colorbox-patched.js' | javascript_include_tag }}
     {{ '/javascripts/3scale_v2.js' | javascript_include_tag }}
     {{ '/javascripts/excanvas.compiled.js' | javascript_include_tag }}
+     {% if provider.payment_gateway_type == :stripe %}
+      {{ 'loadStripe.js' | javascript_include_tag }}
```

Comment:
Not `javascript_pack_tag`?

Reply:
`active_docs` is also using `javascript_include_tag`
https://github.com/3scale/porta/blob/master/lib/developer_portal/app/views/developer_portal/docs.html.liquid#L2

Reply:
`javascript_pack_tag` dosn't work, It is related to the way assets are managed [here](https://github.com/3scale/porta/blob/068c2aff015270507ef1248b45953df3124da72f/lib/developer_portal/lib/liquid/filters/rails_helpers.rb#L49), using `javascript_include_tag`